docs/docs/05-database-fundamentals.md

```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
 ```sql
 TRUNCATE TABLE table_name;
 ```
+
+### [1.4] DCL (Data Control Language)
+เป็นกลุ่มคำสั่งที่จะช่วยให้ผู้บริหารฐานข้อมูล (DBA)
+สามารถควบคุมฐานข้อมูลเพื่อ<u>กำหนดสิทธิการอนุญาต</u> (Grant)  หรือ <u>การยกเลิกการเข้าใช้</u> (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
```

Comment:
Not sure why Markdown doesn't render the underlined text. I guess we'd better use the normal text instead? 

```suggestion
สามารถควบคุมฐานข้อมูลเพื่อกำหนดสิทธิการอนุญาต (Grant)  หรือ การยกเลิกการเข้าใช้ (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
```